phys2bids/utils.py

```diff
@@ -147,36 +147,7 @@ def check_file_exists(filename):
     """
     if not os.path.isfile(filename) and filename is not None:
         raise FileNotFoundError(f'The file {filename} does not exist!')
-
-
-def move_file(oldpath, newpath, ext=''):
-    """
-    Move file from oldpath to newpath.
-
-    If file already exists, removes it first.
-
-    Parameters
-    ----------
-    oldpath: str or path
-        A string or a fullpath to a file that has to be moved
-    newpath: str or path
-        A string or a fullpath to the new destination of the file
-    ext: str
-        Possible extension to add to the oldpath and newpath. Not necessary.
-
-    Notes
-    -----
-    Outcome:
-    newpath + ext:
-        Moves file to new destination
-    """
-    check_file_exists(oldpath + ext)
-
-    if os.path.isfile(newpath + ext):
-        os.remove(newpath + ext)
-
-    os.rename(oldpath + ext, newpath + ext)
-
+        
```

Comment:
Is there a whitespace here?

Reply:
Yes, there were whitespaces. I deleted them using another editor.